torch/fx/experimental/proxy_tensor.py

```diff
@@ -467,7 +468,18 @@ def __sym_dispatch__(self, func, types, args, kwargs):
         # func doesn't have a __torch_function__ that Proxy can interpose, so
         # we gotta do it manually
         n_args, n_kwargs = pytree.tree_map_only(fx.Proxy, lambda p: p.node, (p_args, p_kwargs))
-        n_out = self.tracer.create_node("call_function", func, n_args, n_kwargs)
+        import operator
+        mapped = {
+            operator.mul: torch.ops.math.mul,
+            operator.eq: torch.ops.math.eq,
+            operator.gt: torch.ops.math.gt,
+            operator.lt: torch.ops.math.lt,
+        }
+        if func not in mapped:
+            print(func)
+            raise RuntimeError(f"Don't support {func} on symints")
```

Comment:
This should get fixed at the source. SymDispatchMode should get torch.ops.math calls to begin with. Don't post facto translate them in proxy tensor.